optimizely/event/entity/impression_event.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2019 Optimizely
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from .user_event import UserEvent
+
+
+class ImpressionEvent(UserEvent):
```

Comment:
This classification under entity is vague.

`ImpressionEvent` ideally belongs under event/ and not under event/entity.

Similarly `ConversionEvent` and `UserEvent`

Reply:
I'd recommend putting all 3 of them in a single fine in `optimizely/event/user_event.py`